sunpy/database/tables.py

```diff
@@ -381,10 +381,10 @@ def _from_fido_search_result_block(cls, sr_block, default_waveunit=None):
 
         wavelengths = getattr(sr_block, 'wave', None)
         wavelength_temp = {}
-        if isinstance(wavelength_temp, tuple):
+        if isinstance(wavelengths, tuple):
```

Comment:
This couldn't be right, define \wavelength_tempas a dictionary and then check whether it's atuple` instance???